files/restore_backup.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+
+import os
+import re
+import json
```

Comment:
There is no need to import `json` here.  You actually get it from `ansible.module_utils.basic` and it uses try/except logic to pull in simplejson if available.